packages/datetime/src/common/classes.ts

```diff
@@ -18,6 +18,7 @@ export const DATEPICKER_MONTH_SELECT = "pt-datepicker-month-select";
 export const DATEPICKER_YEAR_SELECT = "pt-datepicker-year-select";
 
 export const DATERANGEPICKER = "pt-daterangepicker";
+export const DATERANGEPICKER_SEQUENTIAL = "pt-daterangepicker-sequential";
```

Comment:
This class still abides by the old `isSequential` naming scheme. Can we change to `pt-daterangepicker-contiguous` or something?